composer/2022_airflow_summit/data_analytics_dag_expansion.py

```diff
@@ -0,0 +1,138 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
add a comment at the top somewhere under the license explaining that this DAG is an expansion of the one found in data_analytics_dag.py and runs a more complex dataproc job found in data_analytics_process_expansion.py

Reply:
Sorry about missing it! I have changed the typo from prcp to snow and added the comment.